test/blackbox/utils.go

```diff
@@ -25,6 +26,7 @@ func executeTestCases(t *testing.T, testCases []TestCase, folder string, platfor
 			h := http.FileServer(http.Dir(testCase.TestdataDir))
 			ts := httptest.NewServer(h)
 			githubEnhancer.GithubBaseURL = ts.URL
+			gitlabEnhancer.GITLAB_BASE_URL = ts.URL
```

Comment:
Sorry but it is really bothering me 😅
Please make it consistent (IDC which convention just stick to one)